alert.html

```diff
@@ -1,84 +1,99 @@
-        <div class="summary__box">
-          <img src="" alt="" class="icon" />
-          <p class="text text--bold"></p>
-          <p class="text text--timestamp"></p>
-        </div>
+        <section class="summary">
+            <h2 class="text__headline2">Yesterday</h2>
+            <div class="summary__box">
+                <img src="" alt="" class="icon" />
+                <p class="text text--bold"></p>
+                <p class="text text--timestamp"></p>
+            </div>
 
-        <div class="summary__box">
-          <img src="" alt="" class="icon" />
-          <p class="text text--bold"></p>
-          <p class="text text--timestamp"></p>
-        </div>
+            <div class="summary__box">
+                <img src="" alt="" class="icon" />
+                <p class="text text--bold"></p>
+                <p class="text text--timestamp"></p>
+            </div>
 
-        <div class="summary__box">
-          <img src="" alt="" class="icon" />
-          <p class="text text--bold"></p>
-          <p class="text text--timestamp"></p>
-        </div>
-      </section>
+            <div class="summary__box">
+                <img src="" alt="" class="icon" />
+                <p class="text text--bold"></p>
+                <p class="text text--timestamp"></p>
+            </div>
+        </section>
     </main>
 
     <footer class="footer">
-      <nav class="nav__footer">
-        <!--img src?"#""-->
-        <ul class="navbar__list">
-          <li class="navbar__items">
-            <a href=""><img src="" alt="" class="navbar__image" /></a>
-          </li>
-          <li class="navbar__items">
-            <a href=""><img src="" alt="" class="navbar__image" /></a>
-          </li>
-          <li class="navbar__items">
-            <a href=""><img src="" alt="" class="navbar__image" /></a>
-          </li>
-          <li class="navbar__items">
-            <a href=""><img src="" alt="" class="navbar__image" /></a>
-          </li>
-          <li class="navbar__items">
-            <a href=""><img src="" alt="" class="navbar__image" /></a>
-          </li>
-        </ul>
-      </nav>
+        <nav class="nav__footer">
+            <!--img src?"#""-->
+            <ul class="navbar__list">
+                <li class="navbar__items">
+                    <a href=""><svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M4.29289 5H20.2929M15 11H20.2929M15 15H20.2929M15 19H20.2929M4 12C4 11.7348 4.10536 11.4804 4.29289 11.2929C4.48043 11.1054 4.73478 11 5 11H11C11.2652 11 11.5196 11.1054 11.7071 11.2929C11.8946 11.4804 12 11.7348 12 12V18C12 18.2652 11.8946 18.5196 11.7071 18.7071C11.5196 18.8946 11.2652 19 11 19H5C4.73478 19 4.48043 18.8946 4.29289 18.7071C4.10536 18.5196 4 18.2652 4 18V12Z" stroke="#727477" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+              </svg>
+              </a>
```

Comment:
wie hast du das gemacht? hast du hier die runtergeladenen .svg, die lokal auf deinem rechner liegen eingebunden? ich bekomme das einfach nicht hin... 
please give me a hint! :-)

Reply:
Ich habe die zwei Ordner in den Image Ordner reingepackt.  Und ich dacht ihr bekommt es auch, wenn ich  alles pushe und commit mache. :)

Reply:
ja, das habe ich gesehen. der liegt aber in deinem branch und soweit ich weiß, kann ich darauf nicht zugreifen, solange unsere branches nicht mit main gemerged sind. und das sollten wir erst tun, wenn alle wieder beisammen sind.
was ich aber eigentlich wissen wollte ist, wie ich meinem <svg> tag sage, dass diese bei mir im ordner liegende bilddatei .svg das <svg> sein soll und woher bekommt css die daten für style und viewbox und woher das <<path> tag innerhalb des <svg>. das verstehe ich nicht. geht bei mir nur wenn ich iwo online den svg tag kopiere. ich verstehe nicht, wie das mit einer datei lokal auf meinem rechner geht... :-(

Reply:
on vor den tag stehen eigentlich jewils html-tags. die werden nicht angezeigt... also wie sage ich das dem svg-tag?

Reply:
ich habe es gefunden...! danke

Reply:
ich bin in den Order, habe mir die svg gegriffen mit der richtigen Farbe und dann copy and paste gemacht. Die styles sind mit dabei gewesen.